src/chart/sankey/SankeyView.ts

```diff
@@ -231,7 +231,9 @@ class SankeyView extends ChartView {
             const focus = emphasisModel.get('focus');
             toggleHoverEmphasis(
                 curve,
-                focus === 'adjacency' ? edge.getAdjacentDataIndices() : focus,
+                focus === 'adjacency' ? edge.getAdjacentDataIndices()
+                : focus === 'fullPath' ? edge.getFullPathDataIndices()
+                : focus,
```

Comment:
Since we are using this logic for multiple times, please consider making a function to reuse.

Reply:
node and edge function look similar but invoke different function.